core/unit_test/tools/TestBuiltinTuners.cpp

```diff
@@ -93,6 +93,10 @@ int main(int argc, char* argv[]) {
     end_context(kernel_context);
     begin_context(kernel_context);
     set_input_values(kernel_context, 1, &md_kernel_value);
+    auto begin_point = new_team_tuner.get_point(0.0, 0.0, 0.0);
+    assert(std::get<0>(begin_point) == 1);
+    auto end_point = new_team_tuner.get_point(0.9, 0.0, 0.0);
+    assert(std::get<0>(end_point) == 5);
```

Comment:
Would you mind explaining why we expect the returned value to be 1 and 5 respectively?

Reply:
0.0 of the way through the outermost dimension of this tuner is the first option (1). 0.9 of the way through the outermost dimension is 5

Reply:
cfe8be9